Sources/ArcGISToolkit/Components/FeatureFormView/EmbeddedFeatureFormView.swift

```diff
@@ -68,17 +77,25 @@ extension EmbeddedFeatureFormView {
     /// Makes UI for a form element.
     /// - Parameter element: The element to generate UI for.
     @ViewBuilder func makeElement(_ element: FormElement) -> some View {
```

Comment:
I suggest renaming this to something like `section(for:)`. It also doesn't need the `ViewBuilder` attribute.